.github/workflows/build_docs.yml

```diff
@@ -16,24 +17,22 @@ on:
   workflow_dispatch:
 
   schedule:
-    - cron: '00 0 * * *'
+    - cron: "00 0 * * *"
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
 jobs:
-
   build_and_publish_documentation:
-
     # prevent cron jobs from running on forks
     if: |
       (github.event_name == 'schedule'
        && github.repository == 'spectrochempy/spectrochempy') || (
        github.event_name != 'schedule')
 
     name: Build and publish documentation
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```

Comment:
just for my information: would it work with ubuntu-latest now ?

Reply:
I think so. It is just that as it worked on my mac i tried also on github but without success with this change only

Reply:
I put it back to ubuntu... just to be sure ;)